cpp/include/cugraph/algorithms.hpp

```diff
@@ -1587,22 +1587,25 @@ uniform_nbr_sample(raft::handle_t const& handle,
  * @param with_replacement boolean flag specifying if random sampling is done with replacement
  * (true); or, without replacement (false); default = true;
  * @param seed A seed to initialize the random number generator
- * @return tuple device vectors (vertex_t source_vertex, vertex_t destination_vertex, weight_t wgt)
+ * @return tuple device vectors (vertex_t source_vertex, vertex_t destination_vertex, weight_t wgt,
+ * edge_t count)
```

Comment:
`wgt` => `weight` better not use abbreviations in the documentation.
And what is `edge_t count`? # of duplicates? I think this needs more explanation.

Reply:
Updated documentation in next push.